.github/workflows/relay_e2e_health.yml

```diff
@@ -0,0 +1,55 @@
+name: Relay e2e health check
+on:
+  schedule:
+    - cron: '0 8 * * *'
+  
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to run the e2e against'
+        required: true
+        default: 'stage'
+        type: choice
+        options:
+          - stage
+          - prod
+          - dev
+
+jobs:
+  relay_health_check:
+    name: 'Relay e2e health check'
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4.0.2
+        with:
+          node-version-file: 'frontend/package.json'
+          cache: 'npm'
+
+      - name: Install Node dependencies
+        run: npm install
+
+      - name: Install Playwright Browsers
+        run: |
+          npm install -D @playwright/test --with-deps
+          npx playwright install
+
+      - name: Run Playwright tests
+        run: |
+          commandenv="${{ inputs.environment != null && inputs.environment || 'stage' }}"
+          E2E_TEST_ENV=$commandenv npx playwright test --grep "@health_check"

```

Comment:
Note: this is the key change from `playwright.yml`.

`playwright.yml`:
```
commandenv=":${{ inputs.environment != null && inputs.environment || 'stage' }}"
npm run test$commandenv
```

This file:
```
commandenv="${{ inputs.environment != null && inputs.environment || 'stage' }}"
E2E_TEST_ENV=$commandenv npx playwright test --grep "@health_check"
```

It seems possible to merge the two using a second input (`inputs.health_check_only`), but isn't necessary.

Reply:
Made changes and the health check is now an input field to playwright.yml.

I still have relay_e2e_health.yml to run the cronjob, could you double check it? I'm not entirely sure how I can test before we merge and wait until tomorrow.